### ISSUE TITLE
suggestion: rename "dual" mode to "online" mode

### ISSUE THREAD

**Issue by mgaved** [mazi-project/portal#122]
At the Volos MAZI project meeting (Ju,y 2018) there was a suggestion to rename "dual mode" to "online mode". This may be better understood, and also complements the alternative "offline mode".

I second this suggestion.

**Comment by panosnethood** [comment]
Agreed. But I think we should keep offering the original "dual" option, that is a MAZI Zone offering Internet access without captive portal. 

The current updated dual mode including the new captive portal functionality was supposed to be the "managed" mode. For now, this means just that the user needs to click on a button to enter but it could include more features like limited bandwidth, etc.

In general, now that the options get richer (e.g., we have two "splash pages", the one that automatically appears and the MAZI Zone's home page, what we called splash page before), we should rethink the whole naming and presentation of these options.

**Comment by mgaved** [comment]
Thanks - it is natural that a project evolves in its lifespan. 

I find "dual" suggests that the user has two different modes to choose from. 

Perhaps the term "internet only" would be more appropriate. But this is just my opinion :-)

**Comment by panosnethood** [comment]
The option "dual" was not "internet only". It was Internet + MAZI Zone with no captive portal. That is, someone needed to actively type the MAZI Zone URL to enter the MAZI Zone, and otherwise would just enjoy Internet access without any inconvenience. 

Not sure what is the appropriate name for this option, and how technically difficult would be to keep offering next to the newly introduced "captive portal" option, but I just want to stress that it is a valid esage option to consider.

**Comment by haniavis** [comment]
I think that it is good and we have agreed that a captive page will appear in all modes just to inform people about the network they connected and what they can do, access the Internet or not, access local applications or not, the domain URL etc.

So even in dual mode, perhaps we should keep the captive page appearing? could be a way to share the URL they should type. In other words, I would say that we should name "online" the previous "dual" mode which now has a captive page appearing, and we will see what we do with the managed mode, probably building on top of this captive page.

**Comment by panosnethood** [comment]
The problem is that the captive portal does not work well for all cases, and there might be always problems with new devices. So, I think we should still allow the option of no captive portal to the administrator.

Btw, I was recently in an easy jet flight and they had a wireless in flight entertainment system, which worked with a URL and no captive portal.
